### DISCUSSION TITLE
Cant find aml launcher settings

### DISCUSSION THREAD

**Discussion by Sabipinek** [X2CommunityCore/xcom2-launcher#301] [Help] [answered]
**AML Version:** 
1.4.1
im new to the launcher because i wanted to play modded xcom 2 and i cannot enter settings of  launcher.exe 
**Attachments**
* *Consider attaching the `AML.log` file from the AML folder*
* *Consider attaching the `settings.json` file from the AML folder*
* *If applicable, add screenshots to help explain your problem.*

**Comment by Iridar** [comment]
They're right here:

https://i.imgur.com/qkk6Wkq.png

**Comment by Sabipinek** [comment]
> They're right here:
> 
> https://i.imgur.com/qkk6Wkq.png

cannot open this screen at all , i get a pop up that steam isnt on

**Comment by Iridar** [comment, marked as answer]
> > They're right here:
> > https://i.imgur.com/qkk6Wkq.png
> 
> cannot open this screen at all , i get a pop up that steam isnt on

Should have mentioned this from the start. See this: https://github.com/X2CommunityCore/xcom2-launcher/wiki/Troubleshooting#please-start-steam-first
